dashboard/test/controllers/levels_controller_test.rb

```diff
@@ -89,9 +89,8 @@ def create_level_with_rubric
       "coursed-2017", "coursee-2017", "coursef-2017", "express-2017", "flappy",
       "frozen", "hourofcode", "jigsaw", "playlab", "pre-express-2017", "starwars"
     ]
-    puts scripts - JSON.parse(@response.body)["scriptOptions"].map {|option| option[0]}
     assert (scripts - JSON.parse(@response.body)["scriptOptions"].map {|option| option[0]}).empty?
-    assert_equal JSON.parse(@response.body)["ownerOptions"].map {|option| option[0]}, ["Any owner"]
+    assert (["Any owner"] - JSON.parse(@response.body)["ownerOptions"].map {|option| option[0]}).empty?
```

Comment:
Could this line (and line 92) be simplified as `an_array.include? a_value`?

Reply:
It could definitely be simplified but in order to unblock the deploy pipeline I'm going to go ahead and merge.